packages/data/lib/repository/tmdb_repository.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:data/service/tmdb_api_service.dart';
+import 'package:data/service/service_payload.dart';
+import 'package:data/utils/api_key_data.dart';
+import 'package:data/utils/constants.dart';
+import 'package:domain/model/tmdb_response.dart';
+import 'package:domain/repository/tmdb_repository.dart';
+
+class TMDBRepositoryImpl implements TMDBRepository {
+  final TmdbApiService<ServicePayload> _apiService;
+
+  const TMDBRepositoryImpl(this._apiService);
+
+  @override
+  Future<TMDBResponse> getCastImage({required int? tMDBId}) async {
+    final response = await _apiService.get(
+      '${UrlConstants.personUrl}/$tMDBId${UrlConstants.imagesUrl}',
```

Comment:
move url forming into UrlConstants. create here static method, which will receive tmdbId as param and return concatenated string